l10n_it_reverse_charge/models/account_invoice.py

```diff
@@ -13,10 +13,16 @@
 class AccountInvoiceLine(models.Model):
     _inherit = "account.invoice.line"
 
+    @api.multi
+    def _set_rc_flag(self, invoice):
+        self.ensure_one()
+        if invoice.type == 'in_invoice':
```

Comment:
come mai solo per questo tipo?

Reply:
@tafaRU da quanto so il reverse charge va applicato solo su acquisti

Reply:
nel README in effeti è scritto così!